tests/components/incomfort/test_init.py

```diff
@@ -0,0 +1,102 @@
+"""Tests for Intergas InComfort integration."""
+
+from datetime import timedelta
+from unittest.mock import MagicMock, patch
+
+from aiohttp import ClientResponseError
+from incomfortclient import IncomfortError
+import pytest
+from syrupy import SnapshotAssertion
+
+from homeassistant.components.incomfort import DOMAIN
+from homeassistant.components.incomfort.coordinator import UPDATE_INTERVAL
+from homeassistant.const import STATE_UNAVAILABLE
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry as er
+from homeassistant.util.dt import utcnow
+
+from .conftest import MOCK_CONFIG
+
+from tests.common import MockConfigEntry, async_fire_time_changed
+
+
+async def test_setup_platforms(
+    hass: HomeAssistant,
+    mock_incomfort: MagicMock,
+    entity_registry: er.EntityRegistry,
+    snapshot: SnapshotAssertion,
+) -> None:
+    """Test the incomfort integration is set up correctly."""
+    entry = MockConfigEntry(domain=DOMAIN, data=MOCK_CONFIG)
+    entry.add_to_hass(hass)
+    await hass.config_entries.async_setup(entry.entry_id)
+    entity_entries = er.async_entries_for_config_entry(entity_registry, entry.entry_id)
+    assert entity_entries
+    for entity_entry in entity_entries:
+        assert entity_entry == snapshot(name=f"{entity_entry.entity_id}-entry")
+        assert hass.states.get(entity_entry.entity_id) == snapshot(
+            name=f"{entity_entry.entity_id}-state"
+        )
```

Comment:
You can use the snapshot_platform helper here

Reply:
I think I did, not sure what you mean

Reply:
Do a global search for `snapshot_platform`